### PR TITLE
New icon: Tildes

### PR DESCRIPTION
**Pull request by m-p-3**
* [x]  I have carefully read and followed [CONTRIBUTING](https://github.com/aegis-icons/aegis-icons/blob/master/CONTRIBUTING.md) document.

**URL of the site:** https://tildes.net/

**Proof of the Aegis-compatible 2FA:** [tildes/views/settings.py#L116](https://gitlab.com/tildes/tildes/-/blob/master/tildes/tildes/views/settings.py#L116)

**Logo / icon info:** https://gitlab.com/tildes/tildes/-/blob/master/tildes/static/safari-pinned-tab.svg, extracted logo from SVG file, using *path > difference* in Inkscape.

**Background color info:** It's from the logo [favicon-32x32.png](https://gitlab.com/tildes/tildes/-/raw/master/tildes/static/favicon-32x32.png) (`#002b36`)

### CODE REVIEW

**Review by krisu5 — CHANGES_REQUESTED** [review]
1. Replace logo with the [original unchanged one](https://gitlab.com/tildes/tildes/-/blob/master/tildes/static/safari-pinned-tab.svg), square is part of the logo as seen in the website.
2. Use icon grid for getting logo to 1:1 square size (green / black dashed line): https://github.com/aegis-icons/aegis-icons/blob/master/_templates/z_icon_grid.svg

File: icons/1_Primary/Tildes.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1024 1024">
+    <circle cx="512" cy="512" r="512" style="fill:#002b36"/>
```
Comment (krisu5) [inline icons/1_Primary/Tildes.svg]: Line 2 & 3: replace 4 space indentation to 3 space.

**Comment by m-p-3** [comment]
Sorry about the mistakes, it's my first contribution here 😬 

> Replace logo with the [original unchanged one](https://gitlab.com/tildes/tildes/-/blob/master/tildes/static/safari-pinned-tab.svg), square is part of the logo as seen in the website.

Do you want to keep the square as an outline, and keep the outline and the logo in white?, while keeping the background transparent so that the circle color is visible in it? Any specific thickness for the square outline?

> Use icon grid for getting logo to 1:1 square size (green / black dashed line): https://github.com/aegis-icons/aegis-icons/blob/master/_templates/z_icon_grid.svg

Sorry, I thought I had to use https://github.com/aegis-icons/aegis-icons/blob/master/_templates/z_icon_grid.svg, will fix that asap.

> Line 2 & 3: replace 4 space indentation to 3 space.

Sorry, I thought I had entered three spaces.., maybe the web-based GitHub editor auto-added one extra space :/

Thanks, and sorry again for asking so many questions..

**Comment by krisu5** [comment]
> Do you want to keep the square as an outline, and keep the outline and the logo in white?, while keeping the background transparent so that the circle color is visible in it? Any specific thickness for the square outline?

Just change the [Safari pinned tab SVG](https://gitlab.com/tildes/tildes/-/blob/master/tildes/static/safari-pinned-tab.svg) from black to white fill and that's enough

> Thanks, and sorry again for asking so many questions..

It's OK :)

**Comment by m-p-3** [comment]
All done, I think that covers it :)

**Review by krisu5 — APPROVED** [review]

**Pull request merged**